tfhe/src/core_crypto/gpu/vec.rs

```diff
@@ -447,6 +447,8 @@ impl<T: Numeric> Drop for CudaVec<T> {
     /// Free memory for pointer `ptr` synchronously
     fn drop(&mut self) {
         for &gpu_index in self.gpu_indexes.iter() {
+            // Synchronizes the device to be sure no stream is still using this pointer
+            synchronize_device(gpu_index);
```

Comment:
Are we mixing PRs?

Reply:
It's based on a previous PR sorry, only the second commit should be looked at here.